api/graphql/tests/snapshots/snap_test_reservation_units.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# snapshottest: v1 - https://goo.gl/zC4yUc
+from __future__ import unicode_literals
+
+from snapshottest import Snapshot
```

Comment:
I guess we cannot test permissions with this or can we?

Reply:
Not with this one. 

Took the snapshot test dep, so it's not so tedious to write asserts. As you probably noticed there is only one super simple test here, i'll write more, but wanted to get this to review so you have time to review it before you go to vacation.

Reply:
yup, this looks nice. Was asking because I'm trying to write some permission related tests atm to gql.